src/framework/global/xmlreader.cpp

```diff
@@ -172,9 +172,9 @@ QString XmlReader::readElementText(ReadStringBehavior behavior)
     return m_reader->readElementText(_behavior);
 }
 
-bool XmlReader::hasError() const
+bool XmlReader::success() const
 {
-    return m_reader->hasError();
```

Comment:
What is the motivation for the renaming?
I believe that the API (name of the methods) is best kept close to the name of the Qt XmlReader methods

Reply:
Because we will write methods, that will return this as a result:

bool foo() {
XmlReader reader;
...
return !reader.hasError();
}

There is a problem. It's very easy to miss "!" before hasError that leads to incorrect behavior. We already faced this problem (thanks to Elnur for fixing it). So Elnur and I decided to rename this method to make it easier to use